Sources/RInAppMessaging/DependencyManager/MainContainer.swift

```diff
@@ -36,7 +36,7 @@ internal enum MainContainerFactory {
             }),
             ContainerElement(type: ConfigurationManagerType.self, factory: {
                 ConfigurationManager(reachability: manager.resolve(type: ReachabilityType.self),
-                                     configurationService: manager.resolve(type: ConfigurationServiceType.self)!,
+                                     configurationService: manager.resolve(type: ConfigurationServiceType.self),
                                      configurationRepository: manager.resolve(type: ConfigurationRepositoryType.self)!,
```

Comment:
nit: is this force unwrap guaranteed to be safe?

Reply:
Yes. The init cannot fail